app/models/job.rb

```diff
@@ -1,2 +1,5 @@
 class Job < ActiveRecord::Base
+  belongs_to :company
+
+  validates :position, :posting_date, :description, :company_id, presence: true
```

Comment:
I left location out because that one seemed like it could be tough occasionally, but the others I was thinking we could/should kind of think of as requirements. Position/Date/Description for sure (if we can't get these from the posting we probably shouldn't be adding them. Company I was right in the middle on... it seems like this is kind of an important field but would probably be the hardest to get systematically. How about if I just take that presence validation out for the time being? Then we can add it back later if this is easier to get than we expected. Work for you?